src/io/GeantMaterial.hh

```diff
@@ -0,0 +1,56 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeantMaterial.hh
+//! \brief Store material information
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "GeantElement.hh"
+#include "GeantGeometryMap.hh"
+#include "base/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Store data of a given material and its elements.
+ *
+ * Used by the GeantGeometryMap class.
+ *
+ * The data is exported via the app/geant-exporter. For further expanding
+ * this struct, add the aproppriate variables here and fetch the new values in
+ * \c app/geant-exporter.cc : store_geometry(...).
+ *
+ * Units are defined at export time in the aforementioned function.
+ */
+
+enum class MaterialState
+{
+    not_defined,
+    solid,
+    liquid,
+    gas
+};
```

Comment:
I've got a similar enum queued up in the `material` branch. Do we want to use that here, or make this one an 'adapter' class? If so it should be `GeantMaterialState`.

Reply:
Geant4 returns an int for its material state, and I just cast it. If we match your enum with Geant's ordering, we can use it without any additional code. Seems simpler.

Reply:
The problem with a direct cast is that it relies on an implementation detail that the enums have identical layouts/ordering. With a cast, you'd never know if we accidentally switched solid/liquid or if Geant4 changed their ordering. You really need a `switch` that uses the relevant type values to be safe; or at least add test cases for each of the enum values.

Reply:
> With a cast, you'd never know if we accidentally switched solid/liquid or if Geant4 changed their ordering
Yes, and since Geant's `GetMaterialState()` literally returns an `int`, I just repeated the ordering. I'll make a switch that is error safe.

Reply:
Done.